tests/unit-tests/internal/emails/test-class-email-settings-tab.php

```diff
@@ -38,21 +38,21 @@ public function testTabContent_WhenCalledWithEmailNotificationSettings_ReturnsCo
 		$email_settings_tab = new Email_Settings_Tab();
 
 		/* Act. */
-		$content = $email_settings_tab->tab_content( 'a', 'email-notification-settings' );
+		$content = $email_settings_tab->get_content( 'email-notification-settings' );
 
 		/* Assert. */
 		self::assertStringContainsString( '<table', $content );
 	}
 
-	public function testTabContent_WhenCalledWithAnotherTab_ReturnsDefaultContent() {
+	public function testTabContent_WhenCalledWithAnotherTab_ReturnsEmptyContent() {
```

Comment:
Now as we test `get_content` it is worth to update test methods' names, starting them with `testGetContent_`.

Reply:
Fixed in https://github.com/Automattic/sensei/pull/6527/commits/307038b11f41a6c5a7f8b754e593f110f7dc07d4.